react/SelectBox/SelectBox.jsx

```diff
@@ -204,30 +263,41 @@ class SelectBox extends Component {
     } = this.props
     const showOverlay = this.state.isOpen && isMobile
     return (
-      <ReactSelect
-        components={{ DropdownIndicator, Option, ...components }}
-        styles={{ ...customStyles(this.props), ...reactSelectStyles }}
-        onMenuOpen={this.handleOpen}
-        onMenuClose={this.handleClose}
-        {...props}
-        className={classNames(
-          {
-            [styles['select__overlay']]: showOverlay,
-            [styles['select--autowidth']]: !fullwidth,
-            [styles['select--fullwidth']]: fullwidth
-          },
-          className
-        )}
-        // react-select temporarily adds className to its innerComponents
-        // but this behavior will soon be removed. For the moment, we
-        // cancel it by setting it to empty string
-        classNamePrefix={classNamePrefix || ''}
-      />
+      <div
+        ref={element => {
+          this.element = element
+        }}
```

Comment:
Why don't we put the ref on the `ReactSelect` directly?

Reply:
We tried but ReactSelect doesn't forward the ref here, so you got the ReactSelect instance which is not a correct React node